BusBoard.ConsoleApp/ApiHelper.cs

```diff
@@ -13,5 +14,30 @@ public static IEnumerable<T> ApiGet<T>(string baseUrl, string resource)
                 .Execute<List<T>>(new RestRequest(resource, DataFormat.Json))
                 .Data;
         }
+
+        public static PostcodeEntry GetPostcodeEntry(string postcode)
+        {
+            return ApiGet<PostcodeEntry>("https://api.postcodes.io/", $"postcodes/{postcode}").First();
+        }
+
+        public static IEnumerable<StopPointEntry.StopPoint> GetNearestStopPoints(PostcodeEntry postcodeEntry)
+        {
+            var resource =
+                $"StopPoint?stopTypes=NaptanPublicBusCoachTram&radius=400&useStopPointHierarchy=false&modes=bus&categories=none&returnLines=false&lat={postcodeEntry.result.latitude}&lon={postcodeEntry.result.longitude}";
```

Comment:
You can split the string over multiple lines: https://stackoverflow.com/questions/1100260/multiline-string-literal-in-c-sharp 🙂